src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -799,9 +799,16 @@ def choose_entry(_term):
             else:
                 return _term['key']
 
-        term_list = [TermObj(**{'term': choose_entry(term),
-                                'count': term['doc_count']})
-                     for term in contents['myTerms']['buckets']]
+        term_list = []
+        for term in contents['myTerms']['buckets']:
+            if 'myProjectIds' in term:
+                project_ids = [bucket['key'] for bucket in term['myProjectIds']['buckets']]
+                term_list.append(TermObj(**{'term': choose_entry(term),
+                                            'count': term['doc_count'],
```

Comment:
These two lines are identical in both branches. See if you can reduce duplication. `**` is your friend, but not in the way you currently use it.

Reply:
Refactored the construction of the term_list list to avoid code duplication.